templates/resources/deploy-task-engine/bin/run-in-container.sh.j2

```diff
@@ -100,9 +100,13 @@ function store_license {
 
 function generate_product_conf {
 
-    HOSTNAME=$(hostname)
-    if [[ ${USE_IP_AS_HOSTNAME,,} == "true" ]]; then
-      HOSTNAME=$(hostname -i)
+    if [[ -z "$OVERRIDE_HOSTNAME" ]]; then
+      HOSTNAME=$(hostname)
+      if [[ ${USE_IP_AS_HOSTNAME,,} == "true" ]]; then
```

Comment:
```suggestion
      if [[ ${USE_IP_AS_HOSTNAME} == "true" ]]; then
```
All the other occurrences as well. Think it is a typo that is existing already.

Reply:
This is not type the `,,` is doing lower case of the variable value in bash.